farm_eggs.farm_quick.inc

```diff
@@ -93,7 +103,7 @@ function farm_eggs_form($form, &$form_state) {
  * Submit function for eggs quick form.
  */
 function farm_eggs_form_submit($form, &$form_state) {
-
+	$timestamp = strtotime($form_state['values']['timestamp']);
```

Comment:
Add a line break and comment above this line, and a line break below it before the next line of code. Indent should be two spaces for consistency.